hooks/useAlgoExplorer.js

```diff
@@ -0,0 +1,96 @@
+import {
+  fetchAlgorandPrice,
+  fetchExplorerAssetInfo,
+  fetchCurrentAssetPrices
+} from 'services/algoexplorer'
+
+import { routeQueryError } from './useAlgodex'
```

Comment:
It is not necessary to add this file since I ported all the hooks into /hooks. Check /hooks/http/useAlgoExplorer.js